invenio_opendata/testsuite/data/cms/cms-tools-vm-image-Run2011A.xml

```diff
@@ -74,7 +74,7 @@
     </datafield>
     <datafield tag="538" ind1=" " ind2=" ">
       <subfield code="a">Use this with a Micro-CernVM available at</subfield>
-      <subfield code="u">http://cernvm.cern.ch/portal/downloads</subfield>
+      <subfield code="w">http://cernvm.cern.ch/portal/downloads</subfield>
```

Comment:
The link at `538__ $u` will appear on the record page after the updates on `opendata.cfg` file (PR 1024)